tests/safety/common.py

```diff
@@ -637,6 +642,26 @@ def test_reset_angle_measurements(self):
     self.assertEqual(self.safety.get_angle_meas_min(), 0)
     self.assertEqual(self.safety.get_angle_meas_max(), 0)
 
+  def test_vehicle_speed_measurements(self):
+    """
+    Tests:
+     - rx hook correctly parses and rounds the vehicle speed
+     - sample is reset on safety mode init
+    """
+    for speed in np.arange(0, 40, 0.5):
```

Comment:
raise the max?

Reply:
raised to 80, that's the max the tesla signal can represent (Nissan is 90 m/s)